test_manage_simulation.py::test_load_bluesky - Failure

When examining the issues relating to https://github.com/alan-turing-institute/dodo/pull/64 , I compared the method for
integration tests with what is being done in [simurgh's travis script](https://github.com/alan-turing-institute/simurgh/blob/ef1d33916acdfa66738b2ffa1f77fd395e7662e0/.travis.yml#L45)
This lead to the discovery of where it appeared to be issues with
`load_scenario()` function in `bluebird`

When running the integration tests locally using docker the following test fails:
```bash
...
tests/integration/test_manage_simulation.py::test_simulation_control PASSED
tests/integration/test_manage_simulation.py::test_load_bluesky FAILED
tests/integration/test_manage_simulation.py::test_load_fail PASSED
...
```
With the accompanying error of:
```bash
if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 500 Server Error: INTERNAL SERVER ERROR for url: http://localhost:5001/api/v1/ic
```
It appears the test expects the file extension to be lowercase but is in fact
upper case as found in `bluesky/scenario/8.SCN`. A change to
`tests/test_simulation_control.py` should resolve this.

Please see this https://github.com/alan-turing-institute/nats/issues/98 where it outlines the fact that although this appears to be a general issue, it does not seem to arise when one does _not_ use the docker builds

--- a/PyDodo/tests/integration/test_flight_level.py
+++ b/PyDodo/tests/integration/test_flight_level.py
@@ -12,7 +12,7 @@ def test_flight_level():
     cmd = reset_simulation()
     assert cmd == True
 
-    resp = load_scenario("scenario/8.scn")
+    resp = load_scenario("scenario/8.SCN")
     assert resp == True
 
     assert current_flight_level("SCN1001") == 6096

--- a/PyDodo/tests/integration/test_manage_simulation.py
+++ b/PyDodo/tests/integration/test_manage_simulation.py
@@ -102,14 +102,14 @@ def test_load_bluesky():
     """
     resp = reset_simulation()
     assert resp == True
-    
-    resp = load_scenario("scenario/8.scn")
+
+    resp = load_scenario("scenario/8.SCN")
     assert resp == True
 
     resp = reset_simulation()
     assert resp == True
 
-    resp = load_scenario("scenario/8.scn", 1.5)
+    resp = load_scenario("scenario/8.SCN", 1.5)
     assert resp == True
 
 
@@ -122,4 +122,4 @@ def test_load_fail():
         load_scenario("")
 
     with pytest.raises(AssertionError):
-        load_scenario("scenario/8.scn", 0)
+        load_scenario("scenario/8.SCN", 0)

